docs/podman-attach.1.md

```diff
@@ -30,6 +34,10 @@ podman attach foobar
 [root@localhost /]#
 ```
 ```
+podman attach -l
```

Comment:
I'd change the -l to --latest just to be abunduntly clear.  I know most folks will use -l, but this won't leave any doubt due to font issues.

Reply:
fixed